src/functions/functionsRouter.ts

```diff
@@ -0,0 +1,689 @@
+import {
```

Comment:
This is the root of the service -- should be Functions only, not FunctionsRouter

Reply:
Yes, I started doing it as I did for Automations service, because previously there was a way to interact with Functions Coordinator contract. I will fix it.